scripts/kind-e2e/lib_operator_deploy_subm.sh

```diff
@@ -188,7 +188,8 @@ function create_subm_cr() {
   sed -i "/spec:/a \ \ brokerK8sCA: $SUBMARINER_BROKER_CA" $cr_file
   sed -i "/spec:/a \ \ ceIPSecPSK: $SUBMARINER_PSK" $cr_file
   # NB: Quoting bool-like vars is required or Go will type as bool and fail when set as env vars as strs
-  sed -i "/spec:/a \ \ ceIPSecDebug: \"$ce_ipsec_debug\"" $cr_file
```

Comment:
Ooops, I might have messed this up (deleted this line) while experimenting with the GitHub GUI for merge conflicts. I will fix it, my bad.

Reply:
Actually I guess that's not so easy since I can't push to @mangelajo's repo. :(